gmsh_utils/gmsh_IO.py

```diff
@@ -327,7 +334,7 @@ def get_num_nodes_from_elem_type(elem_type: int) -> int:
 
     def generate_geometry(self, point_coordinates: Union[List[List[List[float]]], npt.NDArray[np.float64]],
                           extrusion_length: Union[List[float], npt.NDArray[np.float64]], dims: int,
-                          name_label: List[str], mesh_name: str, mesh_size=-1) -> None:
+                          mesh_name: str, name_label: List[str], mesh_size=-1) -> None:
```

Comment:
can you also reorder the docstring. Or dont reorder it here, because its not needed